### ISSUE TITLE
Scala.js tests don't run

### ISSUE THREAD

**Issue by ashawley** [typelevel/scalacheck#557]
There is no output when running tests for Scala.js, and the count of tests is zero.  I can't get a Scala.js to fail, so it doesn't seem they are being run.

```
sbt:scalacheck> js/testOnly org.scalacheck.PropSpecification -- -s 1
[warn] Multiple main classes detected.  Run 'show discoveredMainClasses' to see the list
[info] Passed: Total 0, Failed 0, Errors 0, Passed 0
[success] Total time: 3 s, completed Sep 21, 2019 9:33:01 AM
```

<details>
<summary>
Checking out the 1.14.0_sonatype branch it works again.
</summary>

```
[warn] Multiple main classes detected.  Run 'show discoveredMainClasses' to see the list
[info] Fast optimizing js/target/scala-2.12/scalacheck-test-fastopt.js
[info] + Prop.lzy: OK, proved property.
[info] + Prop.Prop.++ False: OK, passed 1 tests.
[info] + Prop.someFailing: OK, passed 1 tests.
[info] + Prop.Prop.&& Exception: OK, passed 1 tests.
[info] + Prop.proved: OK, passed 1 tests.
[info] + Prop.Prop.&& Identity: OK, passed 1 tests.
[info] + Prop.atLeastOne: OK, passed 1 tests.
[info] + Prop.undecided: OK, passed 1 tests.
[info] + Prop.exception: OK, passed 1 tests.
[info] + Prop.all: OK, passed 1 tests.
[info] + Prop.Prop.++ Identity 1: OK, passed 1 tests.
[info] + Prop.Prop.&& Right prio: OK, passed 1 tests.
[info] + Prop.delay: OK, proved property.
[info] + Prop.secure: OK, passed 1 tests.
[info] + Prop.Prop.==> short circuit: OK, passed 1 tests.
[info] + Prop.Prop.++ Exception: OK, passed 1 tests.
[info] + Prop.Prop.==> true: OK, passed 1 tests.
[info] + Prop.Prop.&& Undecided: OK, passed 1 tests.
[info] + Prop.prop.useSeed is deterministic: OK, passed 1 tests.
[info] + Prop.Prop.&& Exception 2: OK, passed 1 tests.
[info] + Prop.Prop.|| Commutativity: OK, passed 1 tests.
[info] + Prop.Prop.|| Exception: OK, passed 1 tests.
[info] + Prop.throws: OK, proved property.
[info] + Prop.Prop.==> undecided: OK, passed 1 tests.
[info] + Prop.Prop.&& Commutativity: OK, passed 1 tests.
[info] + Prop.Prop.|| True: OK, passed 1 tests.
[info] + Prop.prop.useSeed is deterministic (pt. 2): OK, passed 1 tests.
[info] + Prop.sizedProp: OK, passed 1 tests.
[info] + Prop.Prop.|| Undecided: OK, passed 1 tests.
[info] + Prop.Gen.Parameters.withInitialSeed is deterministic: OK, passed 1 tests.
[info] + Prop.passed: OK, passed 1 tests.
[info] + Prop.Prop.&& False: OK, passed 1 tests.
[info] + Prop.Prop.==> exception: OK, passed 1 tests.
[info] + Prop.falsified: OK, passed 1 tests.
[info] + Prop.Prop.++ Identity 2: OK, passed 1 tests.
[info] + Prop.noneFailing: OK, passed 1 tests.
[info] + Prop.Prop.|| Identity: OK, passed 1 tests.
[info] + Prop.Prop.++ Commutativity: OK, passed 1 tests.
[info] Passed: Total 38, Failed 0, Errors 0, Passed 38
[success] Total time: 200 s (03:20), completed Sep 21, 2019 9:41:46 AM
```

</details>

**Comment by ashawley** [comment]
I thought it was going to be an issue with the Scala.js test interface from an update to Scala.js, but no.

Bisecting this, it looks like it is caused by the change to `ScalaCheckFramework.scala` in #522, in particular commit ff83d14.

**Comment by non** [comment]
This was fixed and released in 1.14.2.
